docs/src/components/Tabs.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import PropType from 'prop-types';
+
+export const TabItem = (props) => {
+	return props.children;
+};
+TabItem.propTypes = {
+	children: PropType.oneOfType([PropType.string, PropType.array]),
+};
+
+export const Tabs = (props) => {
+	const data = props.children || [];
+	const [visibleTab, setVisibleTab] = useState(0);
+	const listTitles = data.map((item, index) => (
+		<li key={`tab_${item.props.title}`}>
+			<button
+				role="tab"
+				id={`tab_${item.props.title}`}
+				aria-controls={`tabpanel_${item.props.title}`}
+				onClick={() => setVisibleTab(index)}
+				aria-selected={visibleTab === index ? 'true' : 'false'}
+				tabIndex={visibleTab === index ? '-1' : null}
+				className={
+					visibleTab === index ? 'tab-title tab-title--active' : 'tab-title'
+				}>
+				{item.props.title}
+			</button>
+		</li>
+	));
+
+	const listContent = data.map((item, index) => (
+		<div
+			key={`tabpanel_${item.props.title}`}
+			tabIndex="0"
+			role="tabpanel"
+			hidden={visibleTab === index ? null : 'hidden'}
+			id={`tabpanel_${item.props.title}`}
+			style={visibleTab === index ? {} : { display: 'none' }}>
+			{item}
+		</div>
+	));
+	return (
+		<div className="tabs">
+			<ul className="tabs-titles">{listTitles}</ul>
```

Comment:
we want to use a div here instead of a list.  The list implementation actually gives too much information on screenread.

Reply:
updated - also added .DS_Store to the ```gitignore``` files.